content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -62,6 +62,12 @@ backend, is used below.
               requests_per_unit: 100
     {{< /text >}}
 
+1. Create ratelimit service/deployment and redis service/deployment.
```

Comment:
We'd need to emphasize that this is for demo purpose only. I am thinking about something like `Create a global rate limit service which implements Envoy's [rate limit service protocol](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/ratelimit/v3/rls.proto). As a reference, a demo configuration can be found [here](https://github.com/istio/istio/blob/master/samples/ratelimit/rate-limit-service.yaml), which is based on a [reference implementation](https://github.com/envoyproxy/ratelimit) provided by Envoy.`